app/src/main/java/com/duckduckgo/app/generalsettings/GeneralSettingsViewModel.kt

```diff
@@ -154,7 +154,7 @@ class GeneralSettingsViewModel @Inject constructor(
     private fun observeShowOnAppLaunchOption() {
         showOnAppLaunchOptionDataStore.optionFlow
             .onEach { showOnAppLaunchOption ->
-                _viewState.update { it!!.copy(showOnAppLaunchSelectedOption = showOnAppLaunchOption) }
+                _viewState.update { it?.copy(showOnAppLaunchSelectedOption = showOnAppLaunchOption) }
```

Comment:
Sorry if I wasn’t clear — this might update the `_viewState` with a null. What I meant was this:
```suggestion
              _viewState.value?.let { state -> _viewState.update { state.copy(showOnAppLaunchSelectedOption = showOnAppLaunchOption) } }
```